lib/plugin.js

```diff
@@ -115,7 +115,13 @@ internals.schema.strategy = Joi.object({
         timeSkewSec: Joi.number().integer().min(0).default(0)
     })
         .when('.validate', { is: Joi.not(false), then: Joi.allow(false) })
-        .required()
+        .required(),
+
+    headerName: Joi.alternatives()
+        .try(Joi.string(), Joi.boolean()).default('authorization').optional(),
+
+    cookieName: Joi.alternatives()
+        .try(Joi.string(), Joi.boolean()).default('token').optional()
```

Comment:
Why a default value? That does not seem safe.

Reply:
I added a default value for providing a sane default. Having the header named authorization and cookie named token is a common usage that's why I did that. Also it would ensure parity with the current behaviour of using the authorization header

Reply:
This is a breaking change, since existing implementations will suddenly also validate tokens provided in `Cookie: token=<jwt>`.

Reply:
updated this so that by default it only reads from authorization header but not from cookie